css/style.css

```diff
@@ -0,0 +1,203 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  background-image: url(../img/hero-image.jpg);
+  background-position: 5px -744px;
+  background-repeat: no-repeat;
+  width: 1450px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+input {
+  width: 100px;
+  height: 19px;
+  padding: 5px 5px 5px 5px;
+  background: url(../img/search-icon.svg), #707070;
+  background-repeat: no-repeat, no-repeat;
+  background-position: 2px 2px;
+  border-radius: 12px;
+}
+
+input:focus {
+  width: 140px;
+  padding-left: 20px;
+
+  /* margin: 8px 9px; */
+  background: url(../img/search-icon.svg), white;
+  background-repeat: no-repeat, no-repeat;
+  background-position: 2px 2px;
+}
+
+.header ul {
+  list-style-type: none;
+  position: absolute;
+  margin: 0;
+  padding: 0;
+  border-radius: 6px;
+  background: linear-gradient(#9e9d9d, #615e5e, #8d8c8c);
+  width: 881px;
+  left: 17%;
+  top: 105px;
+  height: 36px;
+}
+
+.header li {
+  display: inline;
+  font-family: Arial, Helvetica, sans-serif;
+  border-right: double 1px #8d8c8c;
+  float: left;
+  margin-top: 8px;
+}
+
+footer {
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+  width: 1480px;
+  padding: 1em 0 2.6em;
+  transform: scale(0.8);
+}
+
+footer a {
+  text-decoration: none;
+  color: #08c;
+}
+
+.logo a {
+  padding: 22px 30px 1px 28px !important;
```

Comment:
Please remove the !important keyword it's highly not recommended you can always find a way to fix the selector without using !important keyword.

Reply:
done